webapp/routes/route_team.jsx

```diff
@@ -147,6 +152,63 @@ function onPermalinkEnter(nextState, replace, callback) {
     );
 }
 
+function onChannelByIdentifierEnter(state, replace, callback) {
+    if (state.params.identifier.indexOf('@') === -1) {
+        handleError(state, replace, callback);
+    } else {
+        let username = state.params.identifier;
+        if (username.indexOf('@') === 0) {
+            username = username.slice(1, username.length);
+        } else if (username.indexOf('@') > 0) {
+            username = username.slice(0, username.indexOf('@'));
```

Comment:
This won't work for people where their username doesn't match up with the first half of their email. You'll need to write a new method for the UserStore to get a profile by its username, but there's already a `Client.getByEmail` method to get that part from the server